speechbrain/lobes/models/huggingface_transformers/hubert.py

```diff
@@ -0,0 +1,88 @@
+"""This lobe enables the integration of huggingface pretrained hubert models.
+
+Reference: https://arxiv.org/abs/2006.11477
+Reference: https://arxiv.org/abs/1904.05862
+Reference: https://arxiv.org/abs/2110.13900
+Transformer from HuggingFace needs to be installed:
+https://huggingface.co/transformers/installation.html
+
+Authors
+ * Titouan Parcollet 2021
+ * Boumadane Abdelmoumene 2021
+ * Ha Nguyen 2023
+"""
+
+import logging
+
+from speechbrain.lobes.models.huggingface_transformers.wav2vec import (
+    HuggingFaceWav2Vec2,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class HuggingFaceHuBERT(HuggingFaceWav2Vec2):
```

Comment:
What about changing the name as well ? Makes more sense to call it 'HuBert' than 'HuggingFaceHuBERT' since it is already part of the folder 'huggingface_transformers'. (the comment applies to all the models e.g. wav2vec whisper etc.)